src/profileData/mrepol742.json

```diff
@@ -0,0 +1,10 @@
+{
+  "username": "mrepol742",
+  "name": "Melvin Jones Repol",
+  "image": "https://github-production-user-asset-6210df.s3.amazonaws.com/62317165/292488473-10556d1c-e8e5-4884-af75-781e7df4498e.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20231222%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231222T123824Z&X-Amz-Expires=300&X-Amz-Signature=62d3bfac127ce858d430087d63da9e5ed7471c08df1d083f1178d5dd3a81e3fb&X-Amz-SignedHeaders=host&actor_id=62317165&key_id=0&repo_id=731031630",
```

Comment:
Remove the credentials from the image URL.

Make it as:

"image": "https://github-production-user-asset-6210df.s3.amazonaws.com/62317165/292488473-10556d1c-e8e5-4884-af75-781e7df4498e.png"